pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script runs PypeIt on a pair of MOSFIRE images (A-B)
+"""
+import argparse
+
+from pypeit import msgs
+
+import warnings
+
+def parser(options=None):
+
+    parser = argparse.ArgumentParser(description='Script to run PypeIt on MOSFIRE in A-B mode')
+    parser.add_argument('full_rawpath', type=str, help='Full path to the raw files')
+    parser.add_argument('fileA', type=str, help='A frame')
```

Comment:
This should take a sequence of As and a sequence of Bs. Or better yet, just a sequence of files and you can figure out the sequence from the headers, since they are written there. If the files don't fit into a sequence I think we even know that. The more we can figure out from the headers the better.